src/System.Private.Uri/src/System/Uri.cs

```diff
@@ -2108,13 +2108,21 @@ private unsafe ParsingError PrivateParseMinimal()
                             // Only FILE scheme may have UNC Path flag set
                             _flags |= Flags.UncPath;
                             idx = i;
+
+                            _syntax = IsWindowsSystem ? _syntax : UriParser.UnixFileUri;
```

Comment:
Is it possible to assign here only when not implicit? This would keep explicit \\ paths working as before.